src/libraries/System.DirectoryServices.Protocols/tests/DirectoryServicesTestHelpers.cs

```diff
@@ -31,14 +31,18 @@ public static bool IsLibLdapInstalled
                     }
                     else
                     {
-                        _isLibLdapInstalled = NativeLibrary.TryLoad("libldap-2.4.so.2", out _);
+                        _isLibLdapInstalled =
```

Comment:
This seems related to https://github.com/dotnet/runtime/pull/111553 - was that backported to 9.0?

Reply:
Thanks - reverted these changes.